common/tabs_interference_carousel/reports/pre-a11y-1.json

```diff
@@ -0,0 +1,444 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/conformance-report-en.json",
+
+  "@type": [ "earl:Assertion", "acr:AssessmentReport" ],
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/GCWeb/sites/language/language-en.html",
```

Comment:
Probably this is URL you means
https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/tabs/tabs-carousel-en.html (which seem broken)
or https://wet-boew.github.io/wet-boew/demos/tabs/tabs-carousel-en.html